certora/specs/OperatorRegistryV1_base.spec

```diff
@@ -0,0 +1,120 @@
+import "CVLMath.spec";
+
+using OperatorsRegistryV1Harness as OR;
+
+methods {
+
+    // OperatorsRegistryV1
+    function _.reportStoppedValidatorCounts(uint32[], uint256) external => DISPATCHER(true);
+    function OR.getStoppedAndRequestedExitCounts() external returns (uint32, uint256) envfree;
+    function _.getStoppedAndRequestedExitCounts() external => DISPATCHER(true);
+    function _.demandValidatorExits(uint256, uint256) external => DISPATCHER(true);
+    //function _.pickNextValidatorsToDeposit(uint256) external => DISPATCHER(true); // has no effect - CERT-4615
+
+    function _.deposit(bytes,bytes,bytes,bytes32) external => DISPATCHER(true); // has no effect - CERT-4615 
```

Comment:
The deposit contract isn't a part of the scene, so this summary is vacuous (there are no implementations available).
If you do not care about the outcome/actual functionality, you can summarize with NONDET (as long as it doesn't return dynamic arrays).
General note:
Whenever one summarizes with DISPATCHER(true), it is very important to make sure that there is at least one such implementation in the scene.